docs/source/azure/configuration/containerhosting.rst

```diff
@@ -40,137 +34,48 @@ the policy will be ignored.
-            azure_event_queue_name=custodian-aci-queue \
-            azure_container_storage=https://myStorageAccount.blob.core.windows.net/aci-policies \
-            azure_event_queue_resource_id=/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/my-resource-group/providers/Microsoft.Storage/storageAccounts/myStorageAccount
-
-Kubernetes (Helm Chart)
------------------------
-
-A helm chart is provided that will deploy a set of cloud custodian containers against a set of 
-subscriptions to be monitored. For information on how to customize the values, reference 
-the helm chart's values.yaml.
-
-.. code-block:: yaml
-
-    # sample-values.yaml
-
-    defaultEnvironment:
-      AZURE_TENANT_ID: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
-      AZURE_CLIENT_ID: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
-      AZURE_EVENT_QUEUE_NAME: "cloud-custodian-events"
-    
-    defaultSecretEnvironment:
-      AZURE_CLIENT_SECRET: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
-
-    subscriptionHosts:
-      - name: "my-first-subscription"
-        environment:
-          AZURE_SUBSCRIPTION_ID: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
-          AZURE_CONTAINER_STORAGE: "https://firstStorageAccount.blob.core.windows.net/cloud-custodian-policies"
-          AZURE_EVENT_QUEUE_RESOURCE_ID: "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myResourceGroup/providers/Microsoft.Storage/storageAccounts/firstStorageAccount"
-      - name: "my-second-subscription"
-        environment:
-          AZURE_SUBSCRIPTION_ID: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
-          AZURE_CONTAINER_STORAGE: "https://secondStorageAccount.blob.core.windows.net/more-policies"
-          AZURE_EVENT_QUEUE_RESOURCE_ID: "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myOtherResourceGroup/providers/Microsoft.Storage/storageAccounts/secondStorageAccount"
-
-To deploy the chart:
-
-.. code-block:: bash
+Configuration
+#############
 
-    helm upgrade --install --debug --namespace cloud-custodian --values /path/to/sample-values.yaml my-cloud-custodian-deployment tools/ops/azure/container-host/chart
+Configuration for the container host is provided as environment variables. 
+See :ref:`azure_authentication` for authenticating the contaienr host. 
 
+The identity provided to the container host must have the following roles in azure:
 
-Helm Chart Deployment Script
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- ``Reader`` and ``Storage Blob Data Contributor`` on the Storage Account that holds the policy files.
 
-Additionally, a utility script for deploying the helm chart against either a single subscription 
-or all subscriptions in a management group is provided. When deploying for a management group,
-all of the containers will share the same policy storage and storage account for event queues.
+- ``Reader`` and ``Storage Queue Message Processor`` on the Storage Account that the event queue will live in.
```

Comment:
First mention of the event queue. Should this in the image above?

Reply:
yeah, I should add that to the image.